include/cuda/std/detail/__config

```diff
@@ -45,19 +45,6 @@
 #endif
 
 // request these outcomes
-
-#if defined(__CUDACC_RTC__)
-#  if defined(__CUDACC_RTC_INT128__)
-#    define __SIZEOF_INT128__ 16 // This macro is required in order to use int128 within the library
```

Comment:
Why did this macro exist. 🤔 I feel like this was some hack job leftover that I wrote back when NVCC initially added support.

Reply:
I believe the `__SIZEOF_INT128__ ` macro is a very old one and I did not delete it out of pure paranoia